content/components/action-menu.mdx

```diff
@@ -2,7 +2,7 @@
 title: Action menu
 reactId: action_menu
 figma: https://www.figma.com/file/GCvY3Qv8czRgZgvl1dG6lp/Primer-Web?node-id=17091%3A66640
-description: Action menu is a component based on action list for creating a menu of actions that expands through a trigger button.
+description: Action menu is composed of action list and overlay patterns used for quick actions and selections.
```

Comment:
What do you think about removing reference to action list? It feels like an implementation detail in the context of a generic component description. I think this could be more concise and implementation-agnostic it focuses on use-case.

E.g. 

Action menu is used for quick actions and selections

Also sidenote that the Primer Brand ActionMenu doesn't utilise an ActionList under the hood, but use-case and behaviours are otherwise identical.

Reply:
> Action menu is composed of action list and overlay patterns used for quick actions and selections.

I think this suggestion sounds fine, I'm not sure I agree we should be hiding how things are built, it provides some helpful context.

> Also sidenote that the Primer Brand ActionMenu doesn't utilise an ActionList under the hood, but use-case and behaviours are otherwise identical. Would that then disqualify it from being an ActionMenu.

If the end component is the same general component, we should still be calling this Action menu in Primer brand, even if they're not built the same. I don't see any issue with this. They have different visual executions (e.g. one is for brand) and IMO would make sense that they're built slightly different.